homeassistant/components/smappee/binary_sensor.py

```diff
@@ -142,8 +142,7 @@ def icon(self):
     @property
     def device_class(self):
```

Comment:
Don't implement this at all if we just want to return the default `None`.

Generally it's preferred to set a device class instead of customizing the icon. The icon is a single frontend feature while the device class characterizes the entity in a more general way which we can leverage and build upon to generate many features.